app/services/github_handler.rb

```diff
@@ -40,56 +40,23 @@ def handle
   end
 
   def webhook_verified?
-    digest = OpenSSL::HMAC.hexdigest('SHA1', ENV['GITHUB_ANALYZER_WEBHOOK_SECRET'], raw_payload)
+    digest = OpenSSL::HMAC.hexdigest('SHA1', ENV['GITHUB_ANALYZER_WEBHOOK_SECRET'], payload)
     ActiveSupport::SecurityUtils.secure_compare("sha1=#{digest}", signature)
   end
 
   def opened
-    pr_data = payload.pull_request
-    PullRequest.create!(
-      node_id: pr_data.node_id,
-      number: pr_data.number,
-      state: pr_data.state,
-      locked: pr_data.locked,
-      draft: pr_data.draft,
-      title: pr_data.title,
-      body: pr_data.body,
-      closed_at: pr_data.closed_at,
-      merged_at: pr_data.merged_at,
-      merged: pr_data.merged,
-      github_id: pr_data.id
-    )
+    GithubJobs::OpenedJob.perform_later(payload)
   end
 
   def closed
-    find_pr.closed!
+    GithubJobs::ClosedJob.perform_later(payload)
```

Comment:
how big can these payloads be? Isn't it a problem for Redis?

Reply:
There aren't big, I don't think it would be a problem for the future